_includes/trackers_index_response.json

```diff
@@ -0,0 +1,65 @@
+{
+  "trackers" : [
+    {
+      "id": "4a6922e2-09ad-4724-807c-7b4e572d3c6b",
+      "carrier_tracking_no": "723558934169",
+      "status": "delivered",
+      "created_at": "2015-07-20T09:35:23+02:00",
+      "tracking_status_updated_at": "2015-08-01T11:35:23+02:00",
+      "last_polling_at": "2015-08-03T10:23:45+02:00",
+      "next_polling_at": null,
```

Comment:
Hmm, you're probably right. I've added a timestamp that's 2 hours after `last_polling_at` for both examples